### DISCUSSION TITLE
set the JAVA_HOME variable

### DISCUSSION THREAD

**Discussion by tamerplc** [AsamK/signal-cli#449] [Q&A]
i got this error which JAVA_HOME variable is configured under this Path"C:\Program Files\JAVA\jre1.8.0_271\bin\javaw.exe"
could any one guide me 

ERROR: JAVA_HOME is not set and no 'java' command could be found in your PATH.

Please set the JAVA_HOME variable in your environment to match the
location of your Java installation.

**Comment by sadcat** [comment]
pls set JAVA_HOME="C:\Program Files\JAVA\jre1.8.0_271"
